packages/account-v2/src/utils/connectedAppsUtils.ts

```diff
@@ -1,2 +1,6 @@
+import dayjs from 'dayjs';
+
 export const getFormattedAppScopes = (scopes: string[] | undefined) =>
     scopes?.map(scope => scope[0].toUpperCase() + scope.substring(1).toLowerCase()).join(', ');
+
+export const formatDate = (date: string | null) => dayjs(date).format('YYYY-MM-DD HH:mm:ss');
```

Comment:
why do we accept null for this function?

Reply:
the place where we use this function takes values directly from server response and it can also be of value `null`. Typescript throws error if null types is not added.

Reply:
i think its better to handle null case  where we call this function.